### ISSUE TITLE
[Troubleshoot]: Problems related to the Value-model-SFT task in RECAP

### ISSUE THREAD

**Issue by Yonglild** [RLinf/RLinf#1206]
### Problem description

Referring to the RECAP tutorial, the libero10-sft dataset and the libero10-train dataset (only 560 episodes) were used to train the value-model-SFT task. The loss of the training set could converge well, and the cat_acc_best index reached over 90%. But the metrics of the validation set are very poor:
train/cat_acc_best=93%,   train/loss=0.53
eval/cat_acc_best=18%, eval/loss=5.0

### Configuration YAML file

YAML config file:

You can also paste the full config here.
```
# Usage: bash examples/recap/value/run_value_sft.sh libero_sft_value

defaults:
  - model/value@actor.model
  - training_backend/fsdp@actor.fsdp_config
  - _self_
  - override hydra/job_logging: stdout

hydra:
  run:
    dir: .
  output_subdir: null
  searchpath:
    - file://${oc.env:REPO_PATH}/examples/sft/config/

cluster:
  num_nodes: 1
  component_placement:
    actor,env,rollout: all

runner:
  task_type: sft
  logger:
    log_path: "/extraStore/groupdatahw01/Embodied/2514220/exp/rlinf/"
    project_name: rlinf
    experiment_name: "debug"
    logger_backends: ["tensorboard"]

  # max_epochs: 30000
  max_steps: 30000
  val_check_interval: 3000
  save_interval: 6000
  # resume_dir: "/extraStore/groupdatahw01/Embodied/2514220/exp/rlinf/value_sft/libero_sft_value-260525/260525_value_sft/checkpoints/global_step_18000"


data:
  # Must match Step 1's data.tag; loads meta/returns_{tag}.parquet (null = legacy meta/returns.parquet)
  tag: null

  train_data_paths:
    - dataset_path: "/extraStore/groupdatahw01/Embodied/2514220/dataset/RECAP-Libero10/libero10_task0_sft"
      type: "sft"
      weight: 1.0
      robot_type: "libero"
      model_type: "pi05"
    - dataset_path: "/extraStore/groupdatahw01/Embodied/2514220/dataset/RECAP-Libero10/libero10_task0_train"
      type: "rollout"
      weight: 1.0
      robot_type: "libero"
      model_type: "pi05"

  eval_data_paths:
    - dataset_path: "/extraStore/groupdatahw01/Embodied/2514220/dataset/RECAP-Libero10/libero10_task0_eval"
      max_samples: 10000
      robot_type: "libero"
      model_type: "pi05"

  # ----- dataloader performance -----
  train_num_workers: 8
  eval_num_workers: 4
  prefetch_factor: 4
  persistent_workers: false
  pin_memory: false
  # ----- shared dataset params -----
  include_state: true
  gamma: 1.0
  action_horizon: 10
  normalize_to_minus_one_zero: true
  include_next_obs: false
  action_dim: 7
  robot_type: "libero"
  model_type: "pi05"

  # ----- mixture settings -----
  balance_weights: true
  seed: 42

algorithm:
  adv_type: gae

actor:
  group_name: "ActorGroup"
  training_backend: "fsdp"
  micro_batch_size: 32
  global_batch_size: 256
  seed: 0

  model:
    precision: bf16
    siglip_path: "/extraStore/groupdatahw01/Embodied/open_models/siglip2-so400m-patch14-224"
    gemma3_path: "/extraStore/groupdatahw01/Embodied/open_models/gemma-3-270m"
    tokenizer_path: "/extraStore/groupdatahw01/Embodied/open_models/gemma-3-270m"
    freeze_vlm: false
    action_dim: 7
    action_horizon: 10

  optim:
    lr: 5.0e-5
    value_lr: 1.0e-4
    adam_beta1: 0.9
    adam_beta2: 0.95
    adam_eps: 1.0e-8
    weight_decay: 1e-10
    clip_grad: 1.0
    lr_scheduler: "constant"
    lr_warmup_steps: 500
    total_training_steps: 8000

  fsdp_config:
    strategy: "fsdp"
    sharding_strategy: "no_shard"
    use_orig_params: False
    gradient_checkpointing: False
    mixed_precision:
      param_dtype: ${actor.model.precision}
      reduce_dtype: ${actor.model.precision}
      buffer_dtype: ${actor.model.precision}

reward:
  use_reward_model: False

critic:
  use_critic_model: False

```

### Log file

You can find the log file in `logs/` folder or the $output_dir/$experiment_name folder (defined in the yaml config) if you are using our example scripts.

Log file:

If you cannot find the log, please provide the full log messages here.
```
<paste log here>
```

### Environment

python -V:
uv pip list:
nvidia-smi:
RLinf version:
Docker image tag: 

### Before submitting a new issue...

- [x] Have you checked relevant issues, FAQs (https://rlinf.readthedocs.io/en/latest/rst_source/faq.html), or asked the chatbot at the top right corner of the [documentation page](https://rlinf.readthedocs.io).

**Comment by LiuZhihao2022** [comment]
@Yonglild Hi, thanks for sharing the details!😊

From the results, it seems that the value model may be overfitting. RECAP can easily overfit when the amount of training data is limited, so the model may perform well on the training episodes but lose its ability to estimate values reliably on the eval set.

The eval results are provided as a reference to help check for overfitting and select a proper checkpoint. We suggest trying to train with all 4096 episodes if possible, and then choosing the checkpoint based on the eval performance.
